setup.py

```diff
@@ -91,11 +92,13 @@ def read_version():
         'Changes': github_url + '/blob/master/CHANGELOG.md'
     },
     packages=find_packages('.', exclude=["tests*"]),
-    python_requires='>=3.6, <4',
```

Comment:
I know that 3.6 reached it's EOL. But why 3.6 is not technically supported anymore?

Reply:
we need 3.7 at least for asynccontextmanager support, but i think i can create own realisation for now and return 3.6 support

Reply:
asynch requires python >= 3.7 :c